blueprints/ember-cli-blueprint-test-helpers/index.js

```diff
@@ -1,12 +1,62 @@
 var Promise = require('ember-cli/lib/ext/promise');
+var existsSync = require('exists-sync');
+var fs = require('fs-extra');
+var path = require('path');
+var merge = require('lodash/merge');
```

Comment:
might be due to `npm@3` flattening and some subdependency installing the full `lodash` package